x/oracle/spec/05_events.md

```diff
@@ -0,0 +1,47 @@
+<!--
+order: 5
+-->
+
+# Events
+
+The oracle module emits the following events:
+
+## EndBlocker
+
+| Type                 | Attribute Key | Attribute Value |
```

Comment:
Can we ensure all these events are correct? For example, the message.action values are not correct.

It should be `/<msg-url>`, e.g. `/cosmos.bank.v1beta1.MsgSend`

Reply:
You can execute a `sdk.MsgTypeURL(&msg)` to get the message type url of a message btw 👍

Reply:
Updated msg actions to `/<msg-url>`